cobc/cobc.c

```diff
@@ -3720,16 +3748,17 @@ process_command_line (const int argc, char **argv)
 			}
 			break;
 
-		case 6:
-			/* -fdefaultbyte=<xx> : Default initialization byte */
```

Comment:
if not used anymore and therefore deleted, please note that in the Changelog

Reply:
done